actions/notification_actions.jsx

```diff
@@ -68,17 +68,21 @@ export function sendDesktopNotification(post, msgProps) {
         const member = getMyChannelMember(state, post.channel_id);
         const isCrtReply = isCollapsedThreadsEnabled(state) && post.root_id !== '';
 
-        if (!member || isChannelMuted(member) || userStatus === UserStatuses.DND || userStatus === UserStatuses.OUT_OF_OFFICE) {
+        if (!member || (!persistent && isChannelMuted(member)) || userStatus === UserStatuses.DND || userStatus === UserStatuses.OUT_OF_OFFICE) {
```

Comment:
0/5 This is a comment about the design, but having these bypass mute feels rather obnoxious. Generally, you wouldn't have a channel muted if it would contain notifications like this anyway

Reply:
cc @esethna